examples/tracing/cache_miss.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+#
+# cache_miss.py Example of using BPF perf event.
```

Comment:
This is a good example, but it should be a good tool as well, so this might belong under /tools, with a man page and examples file.